sw/device/silicon_creator/rom/e2e/BUILD

```
@@ -166,13 +166,89 @@ opentitan_functest(
     ],
 )
 
-opentitan_functest(
-    name = "rom_e2e_keymgr_init",
-    srcs = ["rom_e2e_keymgr_init_test.c"],
+[
+    otp_json(
+        name = "otp_json_keymgr_rom_ext{}_meas".format(config["name"]),
+        partitions = [
+            otp_partition(
+                name = "OWNER_SW_CFG",
+                items = {
+                    "OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN": config["value"],
+                },
+            ),
+        ],
+    )
+    for config in [
+        {
+            "name": "",
+            "value": "739",
+        },
+        {
+            "name": "_no",
+            "value": "0",
+        },
+    ]
+]
```

nit: it might be easier to follow if you can define the cases once similar to other tests in this file: 

```suggestion
KEYMGR_INIT_CASES = [
    {
        "name": "true",
         "val": CONST.TRUE,
     },
     {
        "name": "false",
         "val": CONST.FALSE,
     },
     {
        "name": "zero",
         "val": "0x0",
     },
]

[otp_json(
    name = "otp_json_keymgr_init_{}".format(case["name"]),
    partitions = [
        otp_partition(
            name = "OWNER_SW_CFG",
            items = {
                "OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN": case["val"],
            }
        ),
    ],
) for case in KEYMGR_INIT_CASES]
```

---

Done, I recreated the uniterated original test so it could better maintain default behavior.